plugin.video.imdb.trailers/resources/settings.xml

```diff
@@ -3,6 +3,7 @@
 	<category label="30000">
```

Comment:
would be nice to update the settings file to the new format at some point

Reply:
As Leia does not work with the new format, have left with the old format for now. Thanks